### PR TITLE
Setting correct default route.

### PR DESCRIPTION
**Pull request by kdamas93**
First PR.

### CODE REVIEW

**Review by mark2600 — APPROVED** [review]

File: kazoo_documents/src/kzd_rates.erl [hunk 1]
```diff
@@ -394,8 +394,8 @@ set_default_route(Rate, Prefix) ->
 
 -spec default_routes(doc() | integer() | kz_term:ne_binary()) -> kz_term:ne_binaries().
 default_routes(<<Prefix/binary>>) ->
-    [<<"^\\+?", Prefix/binary, ".+\$">>];
+    [<<"^\\+", Prefix/binary, "(\\d*)$">>];
 default_routes(Prefix) when is_integer(Prefix) ->
-    default_routes(kz_term:to_binary(Prefix));
-default_routes(Rate) ->
-    default_routes(prefix(Rate)).
+             default_routes(kz_term:to_binary(Prefix));
```
Comment (jamesaimonetti) [inline kazoo_documents/src/kzd_rates.erl]: Borked formatting is borked?

File: kazoo_documents/src/kzd_rates.erl [hunk 2]
```diff
@@ -394,7 +394,7 @@ set_default_route(Rate, Prefix) ->
 
 -spec default_routes(doc() | integer() | kz_term:ne_binary()) -> kz_term:ne_binaries().
 default_routes(<<Prefix/binary>>) ->
-    [<<"^\\+?", Prefix/binary, ".+\$">>];
+    [<<"^\\+", Prefix/binary, "(\\d*)$">>];
```
Comment (jamesaimonetti) [inline kazoo_documents/src/kzd_rates.erl:397]: Why a capture group?

File: kazoo_documents/src/kzd_rates.erl (hunk 1 above)
Comment (kdamas93) [reply kazoo_documents/src/kzd_rates.erl]: The formatter(make fmt) put at that way.

File: kazoo_documents/src/kzd_rates.erl (hunk 2 above)
Comment (kdamas93) [reply kazoo_documents/src/kzd_rates.erl:397]: It was requested like this : https://2600hz-commercial.atlassian.net/browse/KCRO-39

**Pull request merged**